docs/resources/equinix_fabric_routingprotocol.md

```diff
@@ -0,0 +1,178 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "equinix_fabric_routingprotocol Resource - terraform-provider-equinix"
+subcategory: "Fabric"
+description: |-
+  Fabric V4 API compatible resource allows creation and management of Equinix Fabric connection
+  ~> Note Equinix Fabric v4 resources and datasources are currently in Beta. The interfaces related to equinix_fabric_ resources and datasources may change ahead of general availability. Please, do not hesitate to report any problems that you experience by opening a new issue https://github.com/equinix/terraform-provider-equinix/issues/new?template=bug.md
+---
+
+# equinix_fabric_routingprotocol (Resource)
```

Comment:
Elsewhere, the naming convention is to separate words with underscores.  For this resource & corresponding data sources, why is it `routingprotocol` rather than `routing_protocol`?

Reply:
@sabwong Can you please take care of this?

Reply:
addressed in code and docs.